layouts/partials/components/questions.html

```diff
@@ -0,0 +1,10 @@
+<div class="card-title mb-5">Have any questions?</div>
+  <a href="https://discord.gg/thebalanceffxiv">
+    <div class="flex flex-col md:flex-row items-left md:items-center mb-9">
+      <img class="mr-4 mb-2" src="/theme-assets/the_balance.png">
+      <div class="flex items-center">
+        <img class="mr-2" src="/theme-assets/orange_discord_no_text.png">
+        <span class="text-link-orange">discord.gg/thebalanceffxiv</span>
```

Comment:
aaaah, we need a discord param so we can use that in places like this

Reply:
for now, can it be hard-coded? or the discord link pay change 😮

Reply:
oh you mean a param in like the config.toml or something

Reply:
ill also have to replace the href in the a tags with this new param too maybe?